docs/1_Getting_Started.md

```diff
@@ -45,6 +45,8 @@ Caliper is split into pacakges that are managed by Lerna, a tool for managing Ja
 - Run `npm run repoclean` in Caliper root folder to ensure that all the packages are clean
 - Run `npm run bootstrap` to bootstrap the packages in the Caliper repository. This will install all package dependancies and link any cross dependancies. It will take some time to finish installation. If it is interrupted by `ctrl+c`, please recover the file `package.json` first and then run `npm run bootstrap` again.
 
+Do not run any of the above comands with `sudo`, as will cause the bootstrap process to fail
```

Comment:
Missed these before :)
comands -> commands
as will -> as it will

Reply:
I did go to school, I promise!